NExtends/Primitives/Enum.extensions.cs

```diff
@@ -123,10 +123,10 @@ public static DisplayAttribute GetDisplayAttribute(this Enum value)
 
 			var member = members[0];
 			var attributes = member.GetCustomAttributes(typeof(DisplayAttribute), false);
-			if (attributes.Length == 0)
+			if (attributes.Count() == 0)
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Use '.Any()' to test whether this 'IEnumerable<System.Attribute>' is empty or not. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=csharpsquid%3AS1155)